packages/flutter/test/material/search_test.dart

```diff
@@ -40,6 +40,26 @@ void main() {
     TestDefaultBinaryMessengerBinding.instance!.defaultBinaryMessenger.setMockMethodCallHandler(SystemChannels.platform, null);
   });
 
+  testWidgets('Changing query moves cursor to the end of query', (
+      WidgetTester tester) async {
```

Comment:
see style guide -- if there's a newline after the `(` there should be one before the `)`